src/tests/test_utils.cpp

```diff
@@ -6,8 +6,7 @@
 #include <config/resource.hpp>
 
 AttributeMap fake_map() {
-    std::shared_ptr<ProtoType> proto_ptr =
-        std::make_shared<ProtoType>(std::move(std::string("hello")));
+    std::shared_ptr<ProtoType> proto_ptr = std::make_shared<ProtoType>(std::string("hello"));
```

Comment:
just wondering, why remove the `std::move()`? I think it does seem a bit excessive for a string so 👍.  Thanks!

Reply:
the `std::move` was leading to `moving a temporary object prevents copy elision` warning from `Wall`

Reply:
I see, thank you for helping me understand! 👍